gamla/excepts_decorator_test.py

```diff
@@ -56,3 +56,25 @@ async def slow_identity(x):
         )(1)
         is None
     )
+
+
+def test_try_and_excepts_sync():
```

Comment:
seperate the test cases
make second test case not ignore the exception in the handler, so people will know it receives the exception as the first argument. i.e: 
excepts_decorator.try_and_excepts(
            SomeException,
            gamla.identity,
            functional.make_raise(SomeException),
        )(1) == (SomeException(),1)
        
or something similar